homeassistant/components/xiaomi_miio/humidifier.py

```diff
@@ -205,7 +205,7 @@ def __init__(self, name, device, entry, unique_id, coordinator):
             self._available_modes = AVAILABLE_MODES_CA4
             self._min_humidity = 30
             self._max_humidity = 80
-            self._humidity_steps = 100
+            self._humidity_steps = 10
```

Comment:
This change looks unrelated to the fan changes. Why do we change the steps of the humidifier?

Reply:
Looks like a big, keeping in mind that min humidity is 30 and max is 80. But I can remove this.

Reply:
If it s a bug please make a separate PR with that fix.

Reply:
The CA4 can set humidity with steps of 1%.